app/build.gradle

```diff
@@ -141,17 +141,22 @@ dependencies {
     def junitLib = "junit:junit:4.12"
     def mockitoVersion = "2.13.0"
     def mockitoLib = "org.mockito:mockito-core:$mockitoVersion"
+    def mockitoKotlinLib = "com.nhaarman:mockito-kotlin-kt1.1:1.5.0"
     def supportAnnotationLib = "com.android.support:support-annotations:$supportVersion"
 
     // Unit tests
     testImplementation junitLib
     testImplementation mockitoLib
     testImplementation supportAnnotationLib
+    testImplementation mockitoKotlinLib
 
     // UI tests
     androidTestImplementation junitLib
     androidTestImplementation mockitoLib
     androidTestImplementation supportAnnotationLib
+    androidTestImplementation(mockitoKotlinLib) {
+        exclude module: "kotlin-reflect"
```

Comment:
what's the reason for excluding it here but not in unit test?

Reply:
Not sure exactly why, but it was only causing problems here and not for unit tests, so I opted to minimize the `exclude` usage.